### ISSUE TITLE
Restore the NASA API key link

### ISSUE THREAD

**Issue by jasongrout** [jupyterlab/jupyterlab#7286]
In our extension tutorial in our docs, we have a link to the NASA API key documentation:

https://github.com/jupyterlab/jupyterlab/blob/1e8d871fc707ab4477792966c3e1b91cd780a507/docs/source/developer/extension_tutorial.rst#learn-more

```rst
-  Add a setting for the user to put in their `API key <https://api.nasa.gov/api.html#authentication>`__ so they can make many more requests per hour than the demo key allows.
```

A few days ago, NASA redid their web page and accidentally deleted the documentation we were linking to. I've been corresponding with someone there about this, and they plan to put this back in. For now, I changed this link so that the docs check passes (https://github.com/jupyterlab/jupyterlab/pull/7285/commits/406f30c648d454063076c388d183b0609a87c48c in #7285). When NASA puts that documentation back in, we should change the link back to point to the API key documentation.

**Comment by jasongrout** [comment]
This information is now up at https://api.nasa.gov/#authentication, so this link can be restored in our docs.

**Comment by jasongrout** [comment]
Fixed by #7508
